index.js

```diff
@@ -320,13 +325,93 @@ module.exports = {
     return options;
   },
 
+  _shouldHandleTypeScript(config) {
+      let emberCLIBabelConfig = config['ember-cli-babel'] || {};
+      if (typeof emberCLIBabelConfig.enableTypeScriptTransforms === 'boolean') {
+        return emberCLIBabelConfig.enableTypeScriptTransforms;
+      }
+      let typeScriptAddon = this.parent.addons
+        && this.parent.addons.find(a => a.name === 'ember-cli-typescript');
+      return typeof typeScriptAddon !== 'undefined'
+        && semver.gte(typeScriptAddon.pkg.version, '4.0.0-alpha.0');
+  },
+
+	_buildClassFeaturePluginConstraints(constraints, config) {
```

Comment:
This is indented with a tab, can you swap to two spaces instead?

```suggestion
  _buildClassFeaturePluginConstraints(constraints, config) {
```

Reply:
Not sure how that happened. Will do!